app/views/planning_application/neighbour_responses/new.html.erb

```diff
@@ -48,16 +47,17 @@
   <% end %>
   <%= form.govuk_text_field :name %>
   <%= form.govuk_text_field :email %>
-  <%= form.govuk_select :address, @consultation.neighbours.select(&:persisted?).map(&:address), options: { include_blank: true }, label: { text: "Select an existing neighbour address" } %>
+  <%= form.govuk_select :address, @consultation.neighbours.select(&:persisted?).select(&:selected?).map(&:address), options: { include_blank: true }, label: { text: "Select an existing neighbour address" } %>
```

Comment:
Is this query something we could add to the model as a class method perhaps

`@consultation.neighbours.select(&:persisted?).select(&:selected?).map(&:address)`